src/Components/RightSidebar/RightSidebar.jsx

```diff
@@ -61,7 +76,12 @@ export const RightSidebar = () => {
                     <h1>{user.firstName}</h1>
                     <h3>@{user.username}</h3>
```

Comment:
suggestion: you can make this a component to display a user list to follow

Reply:
Thanks for the suggestion will check this out